test-app.gradle

```
@@ -34,6 +35,11 @@ ext.applyTestAppModule = { Project project, String packageName ->
     applyNativeModulesAppBuildGradle(project, packageName)
 
     def manifestFile = findFile("app.json")
+    if (manifestFile == null) {
+        logger.info("Manifest was not found; test app integration is disabled")
+        return
+    }
```

Is this here so we can run tests?

---

That too, but also to be able to dev `rnta/android` without having a test component consuming it.

---

Can we change "Manifest" to "app.json"? And should this be more strongly worded and be a warning at the very least? As this is a RNTA only use case, it would be really confusing to consumers. Is there some way we can only warn if it's not being run by RNTA contributors and the CI?

---

Updated the message and switched to the `warning` logging level. I think it is possible to detect if we're running under RNTA project. Would you prefer to fail hard if `app.json` is missing for consumers?

---

Did it fail previously? I think we want to keep that behaviour.